yarn.lock

```diff
@@ -2951,7 +2951,7 @@ data-view-byte-offset@^1.0.0:
     es-errors "^1.3.0"
     is-data-view "^1.0.1"
 
-date-fns@*, date-fns@^3.6.0:
+date-fns@*:
```

Comment:
Please regenerate `yarn.lock` so that date functions package is removed as a dependency